src/Nowruz/modules/userProfile/components/profileHeader/desktopHeader.tsx

```diff
@@ -79,7 +82,11 @@ export const DesktopHeader: React.FC<DesktopHeaderProps> = ({
         )}
         {!myProfile && (
           <div className={`${css.actionDiv} right-8 w-fit`}>
-            <Button color="primary" variant="outlined" style={{ flex: '1', height: '40px', fontSize: '14px' }}>
+            <Button
+              color="primary"
+              variant="outlined"
+              style={{ flex: '1', height: '40px', fontSize: '14px', gap: '6px' }}
+            >
```

Comment:
Could you please replace inline style with customStyle and pass a className to this button? I think it has been my mistake